src/main/java/seedu/address/logic/parser/AddItemCommandParser.java

```diff
@@ -38,11 +40,30 @@ public AddItemCommand parse(String args) throws ParseException {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddItemCommand.MESSAGE_USAGE), e);
         }
 
-        int currentStock = parseInt(argMultimap.getValue(PREFIX_CURRENTSTOCK).get());
-        int minimumstock = parseInt(argMultimap.getValue(PREFIX_MINIMUMSTOCK).get());
+        String currentStockValue = argMultimap.getValue(PREFIX_CURRENTSTOCK).get();
+        String minimumStockValue = argMultimap.getValue(PREFIX_MINIMUMSTOCK).get();
+
+        if (currentStockValue.isEmpty()) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                    AddItemCommand.MESSAGE_MISSING_CURRENT_STOCK));
+        }
+
+        if (minimumStockValue.isEmpty()) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                    AddItemCommand.MESSAGE_MISSING_MINIMUM_STOCK));
+        }
+
+        try {
+            currentStock = parseInt(currentStockValue);
+            minimumstock = parseInt(minimumStockValue);
+        } catch (NumberFormatException e) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                    AddItemCommand.MESSAGE_USAGE));
+        }
```

Comment:
You want to handle negative as well ? I might do the same for editStock

Reply:
I think it was in one of the bug report #205

Reply:
Stated below that currentstock and minimumstock should be positive already 
Should handle negative values